### ISSUE TITLE
Questions about transform.py

### ISSUE THREAD

**Issue by Yagun** [lengstrom/fast-style-transfer#137]
Hi,

1. In the **net** function you have this code:

        conv_t3 = _conv_layer(conv_t2, 3, 9, 1, **relu=False**)
        preds = tf.nn.tanh(conv_t3) * **150** + 255./2

What does **150** mean here? I compared your code with other projects. They use relu=True and do not multiply by 150. What is your idea? Is it a faster version of instance normalization?

2. You have a function
_def _instance_norm(net, train=True):_
, but never use the **train** variable. Is it there for a purpose?

**Comment by SunAnLan** [comment]
1. 150 can help converge. if you use 127.5, the tanh must be close to 1 to output a 255, where the derivative is pretty low. 

2. that is quite wired. Maybe the writer tried to make it be like BN layer in contrib

**Comment by suke27** [comment]
for this code
preds = tf.nn.tanh(conv_t3) * 150 + 255./2
I found some algorithm use preds = tf.nn.tanh(conv_t3) * 150 + 255./2 directly
do you compare between with 2 options?
